test/controllers/TopUpPeriodsControllerSpec.scala

```diff
@@ -216,14 +257,11 @@ class TopUpPeriodsControllerSpec extends SpecBaseWithApplication with MockitoSug
 
     "redirect to error page for a POST if dates in furlough and input do not align" in {
 
-      val userAnswers = mandatoryAnswers
-        .setValue(SalaryQuestionPage, Salary(2000))
-
-      val application = applicationBuilder(userAnswers = Some(userAnswers)).build()
+      val application = applicationBuilder(userAnswers = Some(baseUserAnswers)).build()
 
       val request =
         FakeRequest(POST, topupPeriodsRoute)
-          .withFormUrlEncodedBody(("value[0]", dates.head.toString), ("value[1]", "2020-04-30"))
+          .withFormUrlEncodedBody(("value[0]", dates(0).toString), ("value[1]", dates(1).toString), ("value[2]", "2020-05-30"))
```

Comment:
I missed this, could be `headOption.value` or something fun.

Reply:
I agree - in this case I opted not to because it is in a test and is followed later in the line by `dates(1)` - I believe this failure would be sufficient and using `headOption.value` would make it less obvious the similarity between the `dates(0)` and `dates(1)` calls.